docs/product-user-guide/src/main/asciidoc/project-deploying-con.adoc

```diff
@@ -0,0 +1,14 @@
+[id='project-deploying-con_{context}']
+= Deploying a project
+
+The deployment process can vary based on the needs of your infrastructure.
```

Comment:
I try to avoid 'need'. Consider requirements?

Reply:
Good call. Agreed.